deployment/network-operator/README.md

```diff
@@ -183,22 +183,6 @@ helm search repo mellanox/network-operator -l
 
 > __NOTE__: add `--devel` option if you want to list beta releases as well
 
-### Download CRDs for the specific release
```

Comment:
Im thinking we should still keep a section about upgrading CRDs.

we should have two subsections

1. manual upgrade (what we have here)
2. automatic upgrade using helm chart by setting .Values.upgradeCRDs to true

Reply:
I changed this section to keep info about manual process